src/flamegraph/color/mod.rs

```diff
@@ -339,6 +340,19 @@ pub(super) fn color(
         let reverse_name_hash = namehash(name.bytes().rev());
 
         (name_hash, reverse_name_hash, reverse_name_hash)
+    } else if deterministic {
+        use std::hash::Hasher;
+        let prefixed_hash = |name: &str, prefix: u8| {
+            let mut hasher = ahash::AHasher::default();
+            hasher.write_u8(prefix);
+            hasher.write(name.as_bytes());
+            (hasher.finish() as f64 / u64::MAX as f64) as f32
+        };
```

Comment:
Ah, this is more extensive than what I had in mind, and is probably excessive given how hashing works. It should be sufficient (and faster) to literally do:

```rust
let mut hasher = ahash::AHasher::default();
hasher.write(name.as_bytes());
let hash = (hasher.finish() as f64 / u64::MAX as f64) as f32;
hasher.write_u8(0);
let hash2 = (hasher.finish() as f64 / u64::MAX as f64) as f32;
(hash, hash2, hash2)
```

Reply:
Is there a reason you're suggesting using only two values, when this returns a 3-tuple?

Reply:
You can do a third too if you want. I'm using the same value for the two last values just because that's the same as the hashing mode does:
https://github.com/jonhoo/inferno/blob/86302c8ddf57281ba982b0da0a1c01aa87eea7fe/src/flamegraph/color/mod.rs#L341